WordPress/Classes/ViewRelated/Stats/SiteStatsTableViewCells.swift

```diff
@@ -26,3 +26,25 @@ struct LatestPostSummaryRow: ImmuTableRow {
 
     }
 }
+
+struct AllTimeStatsRow: ImmuTableRow {
+
+    typealias CellType = SimpleTotalsCell
+
+    static let cell: ImmuTableCell = {
+        let nib = UINib(nibName: "SimpleTotalsCell", bundle: Bundle(for: CellType.self))
+        return ImmuTableCell.nib(nib, CellType.self)
+    }()
+
+    let action: ImmuTableAction? = nil
+
+    func configureCell(_ cell: UITableViewCell) {
+
+        guard let cell = cell as? CellType else {
+            return
+        }
+
```

Comment:
Well, depending on when you can review https://github.com/wordpress-mobile/WordPress-iOS/pull/10493, it would/can be on `develop` for a _really_ short time. 😄